ecs-cli/modules/commands/cluster/cluster_command.go

```diff
@@ -106,6 +106,10 @@ func clusterUpFlags() []cli.Flag {
 			Usage: "[Optional] Specify the AMI ID for your container instances. Defaults to amazon-ecs-optimized AMI.",
 		},
 		cli.BoolFlag{
+			Name:  command.NoAutoAssignPublicIPAddress,
+			Usage: "[Optional] Do not assign public IP addresses to new instances in this VPC. Unless this option is specified, new instances in this VPC receive an automatically assigned public IP address.",
```

Comment:
Perhaps changing the text to: "[Optional] Do not assign public IP addresses to new instances in this VPC. **By default,** new instances in this VPC receive an automatically assigned public IP address."

It's a bit weird to say the default is false here because it's a boolean flag, if you don't specify it's always false.